tests/e2e/specs/subscriptions/myaccount-renew-subscription.spec.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { merchant, shopper, withRestApi } = require( '@woocommerce/e2e-utils' );
+
+import {
+	RUN_SUBSCRIPTIONS_TESTS,
+	describeif,
+	merchantWCP,
+	paymentsShopper,
+} from '../../utils';
+
+import { fillCardDetails, setupCheckout } from '../../utils/payments';
+
+const productName = 'Subscription for renewal testing';
+const productSlug = 'subscription-for-renewal-testing';
+
+const customerBilling = config.get( 'addresses.customer.billing' );
+
+let subscriptionId;
+
+describeif( RUN_SUBSCRIPTIONS_TESTS )(
+	'Subscriptions > Renew a subscription in my account',
+	() => {
+		beforeAll( async () => {
+			await merchant.login();
+
+			// Create subscription product with signup fee
+			await merchantWCP.createSubscriptionProduct( productName, true );
+
+			await merchant.logout();
+		} );
+
+		afterAll( async () => {
+			// Delete the user created with the subscription
```

Comment:
Shouldn't the subscription product and order be deleted here too?

Reply:
Hey, thanks for reviewing! There will be API in the future which could easily clear any subs, orders... for now deleting them could cause additional time when executing the test suite so if they are not causing problems in tests we are totally fine to leave them there. I'll see with the team where we stand with the API so we could add afterwards subscription and order removal in `afterEach`.

Please let me know if I could merge this or you'll do that, thanks.

Reply:
Thanks for the explanation! It makes sense that deleting the user is important, but deleting orders isn't.